packages/terra-show-hide/tests/jest/ShowHideFocuser.test.jsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import ShowHideFocuser from '../../src/ShowHideFocuser';
+
+jest.mock('uuid', () => ({ v4: () => '00000000-0000-0000-0000-000000000000' }));
+
+describe('ShowHideFocuser', () => {
+  const mockPrefix = 'Patients are requesting greater affordability and efficiency in healthcare.';
+  const mockFocusableText = 'In fact, the U.S. ambulatory surgery center market is expected to see a 6.9 % compound annual growth rate, reaching $33 billion by 2028. (2) Cerner understands the urgency to grow in the ambulatory surgery center market while continuing to deliver excellent care.';
+  // eslint-disable-next-line react/prop-types
+  const MockComponent = ({ prefix, focusableText }) => {
+    const ref = React.useRef(null);
+    return <p><ShowHideFocuser ref={ref} prefix={prefix} focusableText={focusableText} /></p>;
+  };
+
+  afterEach(() => {
+    jest.restoreAllMocks();
+  });
```

Comment:
`restoreAllMocks` only works with spies.

https://jestjs.io/docs/jest-object#jestrestoreallmocks 

You should use `resetAllMocks` instead. However, that should be done once at the end in the `afterAll` function.

Reply:
fixed here: https://github.com/cerner/terra-core/pull/3838/commits/aa41c4b79d15cae9e48fece9b1bcbdc462dbe454